<?xml version="1.0"?>
<launch>
    <arg name="ori_target" default="camera_depth_optical_frame"/>
    <arg name="ori_source" default="world"/>
    <node pkg="all_estimator" type="convert_tf" name="convert_tf">
        <param name="ori_target" value="$(arg ori_target)"/>
        <param name="ori_source" value="$(arg ori_source)"/>
    </node>
</launch>